README.md

```diff
@@ -1616,6 +1616,19 @@ Table of Contents
 
 **[⬆ back to top](#table-of-contents)**
 
+## Game Development
```

Comment:
Should also add it to the top section pls.

Please keep in mind the scope of the list too, this might just be way out of scope


```
The scope of this particular list is limited to things that infrastructure developers (System Administrator, DevOps Practitioners, etc.) are likely to find useful. We love all the free services out there, but it would be good to keep it on topic. It's a bit of a grey line at times so this is a bit opinionated; do not be offended if I do not accept your contribution.```

Reply:
No worries. I just wanted to add some resources which came into my sight.

Reply:
@ripienaar Just added to the top section.